docs/source/operations/projections/lccm.rst

```diff
@@ -0,0 +1,35 @@
+.. _lccm:
+
+********************************************************************************
+Lambert Conformal Conic (2SP Michigan)
+********************************************************************************
+
+.. figure:: ./images/lcc.png
+   :width: 500 px
+   :align: center
+   :alt:   Lambert Conformal Conic (2SP Michigan)
+
+   proj-string: ``+proj=lccm +lon_0=-90 +k=1.0000382``
```

Comment:
should be scale and not k ?
(I was wondering if 'k' shouldn't be used instead of 'scale', but its meaning in LCC_2SP Michigan is very specific, so perhaps scale is fine)

Reply:
Good catch! As I said in the previous comment I tried 'k' first, but then switched to 'scale', but forgot to change the doc.